spec/README.md

```diff
@@ -107,12 +107,22 @@ defined in the default registry.
 > (such as due to the release of new CLDR versions).
 
 Updates to this specification will only reserve, define, or require
-function names or function option names
-consisting of characters in the ranges a-z, A-Z, and 0-9.
-All other names in these categories are reserved for the use of implementations or users.
+function identifiers and function option identifiers
+which satisfy either of the following two requirements:
+- Includes no namespace,
+  and has a name consisting of characters in the ranges a-z, A-Z, and 0-9.
+- Uses the `u:` namespace.
+
+All other identifiers in these categories are reserved for the use of implementations or users.
+
+Updates to this specification will only reserve, define, or require
+function option values
+consisting of characters in the ranges a-z, A-Z, and 0-9,
+and the characters U+002D HYPHEN-MINUS `-` and U+002E FULL STOP `.`.
+All other option values are reserved for the use of implementations or users.
```

Comment:
I don't think this is quite right. The problem is that we already have some options that accept e.g. literal and we want to be somewhat permissive here (notice that you're carving out numeric literals). The real limitation wants to restrict our defining keywords.

Notice that the current policy has **no** promises on option _values_. I think it is hard to restrict "future us" from defining more keywords or special literal restrictions blindly. I don't think we should plunge into it this way. Instead, we should carve out what we know we won't do. Perhaps:

```suggestion
Updates to this specification will not reserve, define, or require
_option_ values in **standard** or **optional** _functions_ that start with the following list of symbols.
These values are reserved for use by implementations or user-defined extensions.
- `\_` LOW LINE
- _something else??_
```

Reply:
I like reserving the u: namespace for Unicode. I would go a bit further in reserving all the single-letter namespaces at least for now, until we have a fuller picture of how to structure them to avoid collisions (both across organizations and across time).

However, I don't like the reservation of option value characters at all, based on the discussion.